src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -217,17 +216,6 @@ public SAMFileHeader getHeaderForReads() {
     }
 
     /**
-     * Returns the header for the specified source of Features
-     *
-     * @param featureDescriptor FeatureInput whose header to retrieve
-     * @param <T> type of Feature in our FeatureInput
-     * @return header for the provided FeatureInput (null if we have no sources of Features)
-     */
-    public <T extends Feature> Object getHeaderForFeatures( final FeatureInput<T> featureDescriptor ) {
-        return hasFeatures() ? features.getHeader(featureDescriptor) : null;
```

Comment:
This is part of the tool API and should not be deleted (even if it's not currently used).